oak-run-commons/src/main/java/org/apache/jackrabbit/oak/index/indexer/document/flatfile/FlatFileSplitter.java

```diff
@@ -73,7 +74,9 @@ public class FlatFileSplitter {
     private Set<String> splitNodeTypeNames;
     private boolean useCompression = Boolean.parseBoolean(System.getProperty(OAK_INDEXER_USE_ZIP, "true"));
     private boolean useLZ4 = Boolean.parseBoolean(System.getProperty(OAK_INDEXER_USE_LZ4, "false"));
-
+    
+    static Predicate IS_SPLIT = (Predicate<File>) path -> path.getParent().endsWith(SPLIT_DIR_NAME);
```

Comment:
I would change this with the following
```suggestion
    static Predicate<File> IS_SPLIT = path -> path.getParent().endsWith(SPLIT_DIR_NAME);
```